code/modules/reagents/chemistry/machinery/chem_master.dm

```diff
@@ -1,5 +1,5 @@
 /obj/machinery/chem_master
-	name = "ChemMaster 3000"
+	name = "ChemMaster 3001"
```

Comment:
Why...?

Reply:
New feature release meant new version, and NT forced all stations to purchase the updated units.

Reply:
u r cringe